docs/README.md

```diff
@@ -102,6 +102,61 @@ The block still doesn't have CSS Handles for its specific customization.
 All CSS Handles available for the Image block are the ones available for the `slider-layout` block. Take a look at the Customization section in the [**Slider Layout documentation**](https://vtex.io/docs/app/vtex.slider-layout).
 Note that the `image-slider` uses our `vtex.slider-layout` app, so all the CSS namespaces defined by it are also available for `image-slider`. Take a look at [Slider-Layout](https://vtex.io/docs/app/vtex.slider-layout).
 
+## Image Protocol Implementation
+
+In the `manifest.json` we added Session Client app and also the Image Protocol app in the dependency list:
+
+```json
+ "dependencies": {
+    "vtex.session-client": "1.x",
+    "vtex.image-protocol": "0.x"
+  }
+```
+The interface `ImageTypes.ts` is modified to have the isMobile and imageProtocolId props:
```

Comment:
@LeslieJRP, I didn't understand this part. Could you help me by answering the following questions?
1. Should the developer add the `ImageTypes.ts` on their code?
2. Is this an example of how should it look? 
3. Is this a step-by-step that the user should follow to modify the `ImageTypes.ts`?
Thanks!

Reply:
Hi @Mariana-Caetano 
1. The developer doesn't need to add anything in the ImageTypes.ts, this is already done
2. The code should be like this, no need to modify anything
3. No, the user doesn't need to modify anything in the code, to modify the props the user can use the json file or the site editor

Hope this helps, let me know if you need any other clarification.
Thanks!